src/bucket.ts

```diff
@@ -113,7 +113,7 @@ export interface EnableLoggingOptions {
 export interface GetFilesOptions {
   autoPaginate?: boolean;
   delimiter?: string;
-  directory?: string;
+  directory?: string; //deprecated
```

Comment:
Is this the recommended approach to declare deprecation in an interface? I don't see another example in the project that deprecates this way, and want to confirm.